src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/SystemService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MarginTrading.OrderbookAggregator.Infrastructure.Implementation
+{
+    internal class SystemService : ISystem
```

Comment:
Quite strange.. what is it for ?

Reply:
This was meant to isolate dependency on the DateTime.UtcNow - for tests.